modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -89,7 +89,12 @@ definition {
 		}
 
 		if (isSet(keywords)) {
-			var body = StringUtil.add("${body},", "\"keywords\": [\"${keywords}\"]", "");
+			if (${keywords} == "") {
+				var body = StringUtil.add("${body},", "\"keywords\": []", "");
+			}
+			else {
+				var body = StringUtil.add("${body},", "\"keywords\": [\"${keywords}\"]", "");
```

Comment:
I'm thinking the else{} might already contain the keyword="", could you check if we can remove the if here?

Reply:
I don't think so, I have tried it, if you use keywords=""
`'var body = StringUtil.add("${body},", "\"keywords\": [\"${keywords}\"]", "");'`
the response is` "keywords" : [""]`

Reply:
Oh, I get it. What about remove the "", this works for example the response is same executing "keywords":[test123] and "keyword":["test123"]. Could you try this on your side see if this works or not? 
`var body = StringUtil.add("${body},", "\"keywords\": [${keywords}]", ""); `

Reply:
"keywords":[test123] this can not work

Reply:
we need "keywords": ["tag1"] and "keywords": []

Reply:
yes, we come to an conclusion that only Integer don't need the "".